src/library/scala/collection/StringOps.scala

```diff
@@ -56,7 +56,7 @@ final class StringOps(private val s: String)
     *                `f` to each element of this String and collecting the results.
     */
   def map(f: Char => Char): String = {
-    val sb = new StringBuilder
+    val sb = new StringBuilder(length)
     for (ch <- s) sb += f(ch)
```

Comment:
Another opportunity: use a `while` loop here rather than the `foreach`

Reply:
yes certainly can be done if we're going for max speedup